src/commands/service/compile.ts

```diff
@@ -11,15 +11,15 @@ import deployer, {createTar} from '../../utils/deployer'
 const ipfsClient = require('ipfs-http-client')
 
 export default class ServiceCompile extends Command {
-  static description = 'Compile a service and upload its source to IPFS'
+  static description = 'Compile a service and upload its source to IPFS server'
 
   static flags = {
     ...Command.flags
   }
 
   static args = [{
-    name: 'SERVICE_PATH_OR_URL',
-    description: 'Path of the service or url to access it',
+    name: 'SERVICE',
```

Comment:
You need to rename `args.SERVICE_PATH_OR_URL` to `args. SERVICE` at line 31 (check for other lines)